### PR TITLE
Refactor/hosting automatize translations

### PR DESCRIPTION
**Pull request by marie-j**
## Refactor/hosting automatize translations

MANAGER-2973

### CODE REVIEW

File: client/app/hosting/cron/hosting-cron.service.js [hunk 1]
```diff
@@ -16,109 +24,118 @@ angular.module('services').service('HostingCron', function hostingCron($q, OvhHt
...
+
+  getAvailableLanguage(serviceName) {
+    return this.OvhHttp.get('/hosting/web/{serviceName}/cronAvailableLanguage', {
+      rootPath: 'apiv6',
+      urlParams: {
+        serviceName,
+      },
+    });
+  }
+
```
Comment (JeremyDec) [inline client/app/hosting/cron/hosting-cron.service.js]:
```suggestion
```

File: client/app/hosting/cron/hosting-cron.service.js [hunk 2]
```diff
@@ -16,109 +24,118 @@ angular.module('services').service('HostingCron', function hostingCron($q, OvhHt
...
+
+  formatLanguage(language) {
+    if (language === OTHER || language === OTHER.toUpperCase()) {
```
Comment (JeremyDec) [inline client/app/hosting/cron/hosting-cron.service.js]: You can simplify the test (this requires to set the test in your test L129) :
```suggestion
    if (language.toUpperCase() === OTHER.toUpperCase()) {
```

File: client/app/hosting/translations/Messages_fr_FR.json [hunk 3]
```diff
@@ -252,42 +252,6 @@
   "hosting_tab_CRON_table_header_command": "Commande",
   "hosting_tab_CRON_table_language_OTHER": "Autre",
   "hosting_tab_CRON_table_language_other": "Autre",
-  "hosting_tab_CRON_table_language_PHP_4": "PHP 4",
```
Comment (FredericEspiau) [inline client/app/hosting/translations/Messages_fr_FR.json]: 🎉

File: client/app/hosting/cron/CRON.html [hunk 4]
```diff
@@ -36,124 +36,58 @@ <h2 data-translate="hosting_tab_menu_crons"></h2>
...
+                        data-title=":: 'hosting_tab_CRON_table_header_status' | translate"
+                        data-property="status">
+                        <span
+                            class="oui-status"
+                            data-ng-class="{
+                                'oui-status_success': $row.status === 'enabled',
+                                'oui-status_error': $row.status === 'disabled',
+                                'oui-status_warning': $row.status === 'suspended'
+                            }"
+                            data-ng-bind="'hosting_tab_CRON_table_status_' + $row.status | translate">
```
Comment (FredericEspiau) [inline client/app/hosting/cron/CRON.html]:
```suggestion
                            data-translate=":: 'hosting_tab_CRON_table_status_' + $row.status">
```

**Comment by marie-j** [comment]
> Could you please resolve existing conflict?
> 
> Thanks!

@antleblanc done :)

**Pull request merged**